package.json

```
@@ -114,7 +114,7 @@
     "@keystonehq/bc-ur-registry-eth": "^0.6.8",
     "@keystonehq/metamask-airgapped-keyring": "0.2.1",
     "@material-ui/core": "^4.11.0",
-    "@metamask/contract-metadata": "^1.31.0",
+    "@metamask/contract-metadata": "1.35.0",
```

I assume this is what you meant to do?

```suggestion
    "@metamask/contract-metadata": "^1.35.0",
```

Making this change then running `yarn yarn-deduplicate` will also get rid of the 2nd copy in our dependency tree